ddtrace/tracer/spancontext.go

```diff
@@ -127,6 +127,11 @@ func (c *spanContext) baggageItem(key string) string {
 	return c.baggage[key]
 }
 
+func (c *spanContext) meta(key string) (val string, ok bool) {
```

Comment:
Does this function need a note that it isn't concurrent safe? Or perhaps it should call RLock on `c.span`?

Reply:
Good catch. I added the RLock.